internal/metrics.go

```diff
@@ -33,12 +36,28 @@ func newMetrics() *metrics {
 		Name: "fleetlock_unlock_request_count",
 		Help: "Number of unlock requests",
 	})
+	totalRequests := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_requests_total",
+		Help: "Number of requests",
+	}, []string{"path"})
+	responseStatus := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "http_response_status",
```

Comment:
http_response_status_total, the value is a count not the status https://prometheus.io/docs/practices/naming/

Reply:
Done.